spec/features/index_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+feature "visitor sees Dives" do
+  scenario "dive is on the page" do
+
+    visit sites_path
+
+    expect(page).to have_content("Dive")
+  end
+end
```

Comment:
we actually should be able to see all of the links to the dive pages as well here, and test accordingly!